src/main/groovy/com/bmuschko/gradle/docker/tasks/container/DockerCreateContainer.groovy

```diff
@@ -110,7 +111,7 @@ class DockerCreateContainer extends DockerExistingImage {
 
     @Input
     @Optional
-    final Property<Map> envVars = project.objects.property(Map)
+    final MapProperty<Object, Object> envVars = project.objects.mapProperty(Object, Object)
```

Comment:
This should define the concrete types. Let's use String/String here and in the code that passes the key/value pairs to Docker Java, we'll want to remove the support for Closures. There's no more need for Closures as the Property takes care of lazy evaluation.

Reply:
I did wonder about the map being untyped, but I originally decided to leave it alone in the event there was an intentional reason for it.